CMakeLists.txt

```diff
@@ -122,6 +122,9 @@ SET(CMAKE_INSTALL_DIR "${_CMAKE_INSTALL_DIR}" CACHE PATH "The directory cmake fi
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+add_library(RapidJSON INTERFACE)
+target_include_directories(RapidJSON INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
I recommend adding an `ALIAS` target too with `::` in the name. Perhaps `RapidJSON::RapidJSON`. This causes CMake to issue better diagnostics.

Reply:
see https://github.com/Tencent/rapidjson/pull/1810